packages/nekoton_bridge/lib/src/bridge_generated.dart

```diff
@@ -1194,6 +1210,22 @@ abstract class NekotonBridge {
   FlutterRustBridgeTaskConstMeta
       get kRemoveTokenWalletMethodAccountsStorageImplConstMeta;
 
+  /// Remove token wallets signatures from account (remove tokens from account aka disable it via slider).
+  /// account_address - address of account
```

Comment:
Seems like this and following method docs with incorrect argument description [].

Reply:
Why? This is the description for removing list of tokens, and there is description for this